features/AssetCategory.feature

```diff
@@ -284,4 +283,102 @@ Feature: AssetCategory
     Then The document "engine-ayse":"asset-category":"littleTruck" content match:
       | parent | null |
 
+  Scenario: Use metadata enum type and link with assetCategory
+    When I successfully execute the action "device-manager/assetCategory":"create" with args:
+      | engineId  | "engine-ayse"  |
+      | body.name | "coloredTruck" |
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse"          |
+      | body.name      | "color"                |
+      | body.valueType | "string"               |
+      | body.valueList | ["red","blue","green"] |
+      | body.mandatory | true                   |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I successfully execute the action "device-manager/assetCategory":"unlinkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+      | body.value | "red"          |
+    When I successfully execute the action "device-manager/assetCategory":"unlinkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+      | body.value | "black"        |
+    Then I should receive an error matching:
+      | status | 400 |
```

Comment:
You should rather test the `error.id` property to ensure it's the correct error who it's returned

Reply:
I don't know that, but I'm agree, it's really better, I will do that.

Reply:
done